consensus/parlia/parlia.go

```diff
@@ -1158,6 +1177,24 @@ func (p *Parlia) Finalize(chain consensus.ChainHeaderReader, header *types.Heade
 			return err
 		}
 	}
+
+	if p.chainConfig.IsOnFeynman(header.Number, parent.Time, header.Time) {
+		err := p.initializeFeynmanContract(state, header, cx, txs, receipts, systemTxs, usedGas, false)
```

Comment:
1.update system contract
2.run system contract: slash, distribute income, distribute FF reward
3.init Feynman Contract
Is this sequence reasonable? why not run step 3 right after step 1?

Reply:
We did not give much consideration about the sequence, but append new tx at the end.

Reply:
You're right. Fixed